### PR TITLE
187118505 tc data importer qa

### PR DESCRIPTION
**Pull request by ttoomey**
[//]: # 'remove this if PR is for a release-* branch'
# _Please squash merge this PR_

## Description
[PT Issue](https://www.pivotaltracker.com/story/show/187118505)

* add meal service importer
* extrapolate missing enrollment identifiers for services
* adjust custom assessment importers to find-or-create form definitions
* match form definition identifiers to what's defined on staging
* fix missing parse_duration that got lost in the last merge
* normalize all uuids to correct format
* correct fields on case management assessment loader, critical document loader
* adjust custom service loader to be safer when clobbering old data with multi-part imports
* adjust custom service loader "substance abuse" services
* remove unused EHV importer
* correct col names in MHMR  imports
* add rake task to collect quick stats on hud tables as a proof against accidental changes

misc:
* add comment for Rails 7 adjustment to data loader

## Type of change
[//]: # 'remove options that are not relevant'
- [x] Bug fix
- [x] New Feature

## Checklist before requesting review
- [x] I have performed a self-review of my code
- [x] I have run the code that is being changed under ideal conditions, and it doesn't fail
- [x] My code includes comments and/or descriptive variable names to help other engineers understand the intent (or not applicable)
- [x] My code follows the style guidelines of this project (rubocop)
- [x] I have updated the documentation (or not applicable)
- [x] If it's not obvious how to test this change, I have provided testing instructions in this PR or the related issue

### CODE REVIEW

File: drivers/hmis_external_apis/app/models/hmis_external_apis/tc_hmis/importers/loaders/custom_services_loader.rb [hunk 1]
```diff
@@ -427,7 +433,7 @@ def row_enrollment_id(row)
           },
         },
         'Individual TBSS' => {
-          service_type: 'Individual TBSS',
+          service_type: 'TBSS',
```
Comment (gigxz) [inline drivers/hmis_external_apis/app/models/hmis_external_apis/tc_hmis/importers/loaders/custom_services_loader.rb:436]: @ttoomey I pushed a small change to 2 service type names per [request](https://www.pivotaltracker.com/n/projects/2591838/stories/187215113) https://github.com/greenriver/hmis-warehouse/pull/4101/commits/d9d8984b4c1c31ddd4ae3782e12d2c6f385fb7fc

File: drivers/hmis_external_apis/app/models/hmis_external_apis/tc_hmis/importers/loaders/custom_assessment_loader.rb [hunk 2]
```diff
@@ -153,6 +155,15 @@ def create_form_processor_records(rows)
       ar_import(processor_model, records)
     end
 
+    def form_definition
+      @form_definition ||= Hmis::Form::Definition.where(identifier: form_definition_identifier).first_or_create! do |definition|
+        definition.title = form_definition_identifier.humanize
+        definition.status = 'draft'
+        definition.version = 0
+        definition.role = 'FORM_DEFINITION'
```
Comment (gigxz) [inline drivers/hmis_external_apis/app/models/hmis_external_apis/tc_hmis/importers/loaders/custom_assessment_loader.rb:163]: this isn't valid, but we shouldn't hit this code

**Review by gigxz — APPROVED** [review]
Looks great!!! 👏 🌟 🏃‍♂️  let's give it a go

File: drivers/hmis_external_apis/app/models/hmis_external_apis/tc_hmis/importers/loaders/custom_assessment_loader.rb (hunk 2 above)
Comment (ttoomey) [reply drivers/hmis_external_apis/app/models/hmis_external_apis/tc_hmis/importers/loaders/custom_assessment_loader.rb:163]: Oh yeah, this should be 'CUSTOM_ASSESSMENT', thanks!

**Pull request merged**